ContentForm.HTML

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
Formatting makes this file hard to read. Consider auto-formatting. Am sure VS Code could sort this out for you